networks/basic-rust.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
 spec:
   replicas: 2
   ceramic:
+    - env:
```

Comment:
The `ceramic` section is a list, so this isn't quite correct. This section currently is specifying 3 different node configs, two with the `dev` js-ceramic image and the default IPFS image (which is Kubo I think), and one with the rust-ceramic image and the default js-ceramic image (latest). We should only keep one section (just one hyphen).

Reply:
When I just include one hyphen the ceramic (container is ceramic) image that ceramic-0-0 is running is ceramicnetwork/js-ceramic:dev. 
And what ceramic-1-0 pod's ceramic container is running is : ceramicnetwork/js-composedb:latest. 

Let me know if there is a better way to pass the images between both the ceramic containers in ceramic-0-0 and ceramic-1-0